stable/insights-agent/CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 # Changelog
+## 2.6.11
+* Update `nova` version to `v3.4` 
+* Add new flags to `nova find` command 
```

Comment:
An non-blocking nit: mentioning what the new flag is/does will help the reader know how nova may behave differently.

Reply:
done